e2e/__tests__/noTestsFound.test.ts

```diff
@@ -17,7 +17,10 @@ describe('No tests are found', () => {
       '/non/existing/path/',
     ]);
 
-    expect(stdout).toMatch('No tests found');
```

Comment:
I've expanded these assertions to include the exit code and the line about `passWithNoTests`